pkg/main/main.go

```diff
@@ -29,6 +29,8 @@ import (
 	_ "sigs.k8s.io/controller-runtime/pkg/healthz"
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
 
+	hrscheme "github.com/fluxcd/helm-operator/pkg/client/clientset/versioned/scheme"
```

Comment:
I think this should be `hrscheme "github.com/fluxcd/helm-operator/pkg/apis/helm.fluxcd.io/v1`